enhancements/installer/kubevirt-ipi.md

```diff
@@ -0,0 +1,346 @@
+
+##### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+
+##### Tech Preview -> GA 
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+
+**For non-optional features moving to GA, the graduation criteria must include
+end to end tests.**
+
+##### Removing a deprecated feature
+
+TODO
+  - Announce deprecation and support policy of the existing feature
+  - Deprecate the feature
+
+### Upgrade / Downgrade Strategy
+
+TODO
+  If applicable, how will the component be upgraded and downgraded? Make sure this
+  is in the test plan.
+
+
+  Consider the following in developing an upgrade/downgrade strategy for this
+  enhancement:
+  - What changes (in invocations, configurations, API use, etc.) is an existing
+    cluster required to make on upgrade in order to keep previous behavior?
+  - What changes (in invocations, configurations, API use, etc.) is an existing
+    cluster required to make on upgrade in order to make use of the enhancement?
+
+### Version Skew Strategy
+
+TODO
+  What are the guarantees? Make sure this is in the test plan.
+
+  Consider the following in developing a version skew strategy for this
+  enhancement:
+  - During an upgrade, we will always have skew among components, how will this impact your work?
+  - Does this enhancement involve coordinating behavior in the control plane and
+    in the kubelet? How does an n-2 kubelet without this feature available behave
+    when this feature is used?
+  - Will any other components on the node change? For example, changes to CSI, CRI
+    or CNI may require updating that component before the kubelet.
+
+## Implementation History
+
+June 12th 2019 - Presented a fully working POC
+
+## Drawbacks
+
+The idea is to find the best form of an argument why this enhancement should _not_ be implemented.
+
+## Alternatives
+
+Similar to the `Drawbacks` section the `Alternatives` section is used to
+highlight and record other possible approaches to delivering the value proposed
+by an enhancement.
+
+## Infrastructure Needed [optional]
+
+- CI
+Running and end-to-end job is a must for this feature to graduate, and it is a
+non trivial task. oVirt is not a cloud solution and we need to provide a setup
+for a job invocation. We started with deploying a static oVirt deployment on GCP
+and it is working and able start an installation which is initiated from outside
+of GCP. Now we need to make sure the CI job can do the same.
+What's left is to make sure this instance network setup can work with the floating
+IP's we assign to it (the DNS and API VIPS). Currently we assume we can make that
+work because we control the dnsmasq inside VMs network.
+
+What could go wrong?
+- we may not be able to make the CI play nicely on time and we need as much help
```

Comment:
CI is a requirement for going GA with any OCP features. No exceptions, AFAIK.

Reply:
you right, fixing this one